website/static/js/pages/login-page.js

```diff
@@ -20,6 +23,45 @@ var activateToggleBox = function () {
     }
 };
 
+
+var InstitutionViewModel = function() {
+    var self = this;
+    self.inst_names = ko.observableArray([]);
+    self.insts = {};
+    $osf.ajaxJSON(
+        'GET',
+        window.contextVars.apiV2Prefix + 'institutions/',
+        {
+            isCors: true
+        }
+    ).done( function(response){
+        for (var i = 0; i < response.data.length; i++){
+            var name = response.data[i].attributes.name;
+            self.inst_names.push(name);
+            self.insts[name] = response.data[i].attributes.auth_url;
+        }
+    }).fail(function(response){
+    });
+
+    self.instLogin = function(){
+        var key = $('#selected_inst').val();
```

Comment:
Prefer data-binding to reading from the DOM like this. You should be able to remove those CSS IDs and use an observable instead.